### DISCUSSION TITLE
How to filter search results by created date

### DISCUSSION THREAD

**Discussion by nicolesherman00** [microsoft-search/pnp-modern-search#3129] [Q&A] [answered]
We have a news search page that filters by hub site.
We are trying to filter out any news that is older than 3 weeks old. I am not sure I know the best way to achieve that with these webparts  and would love some input.

We are pulling all news from into the search results with the DepartmentId: so that we get all news from all sites in the hub, and promotedstate: 2.

Any suggestions would be greatly appreciated!

**Comment by kasperbolarsen** [comment]
If you plan to be able to filter on the created or modified date fields you will have to map the ows_[fieldname] to a RefinableDate and use that property in your search query and in the refiners.
If the results seems weird I will suggest using the debug layout or SharePoint Search Query tool to verify your data.

**Reply by nicolesherman00** [reply]
How do you tell which field it is to map, if there is so many fields mapped to "created"?
![image](https://github.com/microsoft-search/pnp-modern-search/assets/22782498/759f9788-1cdb-4916-ae47-838e63ba690f)

**Comment by kasperbolarsen** [comment, marked as answer]
If you look at your Managed Properties you will find "Created"
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/3785da01-47d2-46e4-9115-ef1fbab9c8ae)

Normally you will have to map the crawled properties to a RefinableDate in order to use it in a PnP Modern Search Filter web part, but Created and LastModifiedTime has been mapped already by MS  (note the value in the "REFINE" column) .
All you need is to update the query text with  "Created>={today-21}" and only items created within the last 21 days will appear :-)

**Reply by nicolesherman00** [reply]
This worked beautifully. Thank you so much!!
